applications/FluidDynamicsApplication/python_scripts/adjoint_monolithic_solver.py

```diff
@@ -7,17 +7,79 @@
 # Import base class file
 from KratosMultiphysics.FluidDynamicsApplication.adjoint_fluid_solver import AdjointFluidSolver
 
-def CreateSolver(main_model_part, custom_settings):
-    return AdjointVMSMonolithicSolver(main_model_part, custom_settings)
+class StabilizedAdjointFormulation(object):
+    """Helper class to define stabilization-dependent parameters."""
+    def __init__(self,settings):
+        self.element_name = None
+        self.element_integrates_in_time = False
+        self.element_has_nodal_properties = False
+        self.process_data = {}
+
+        if settings.Has("element_type"):
+            formulation = settings["element_type"].GetString()
+            if formulation == "vms":
+                self._SetUpClassicAdjointVMS(settings)
+            elif formulation == "qsvms":
+                self._SetUpAdjointQSVMS(settings)
+        else:
+            print(settings)
+            raise RuntimeError("Argument \'element_type\' not found in stabilization settings.")
+
+    def SetProcessInfo(self,model_part):
+        for variable,value in self.process_data.items():
+            model_part.ProcessInfo[variable] = value
+
+    def _SetUpClassicAdjointVMS(self,settings):
```

Comment:
I'd throw a deprecation warning if this is instantiated. At the end we want to promote the usage of the new implementation.

Reply:
done 👍